cli/src/cli-cmd-parser.c

```diff
@@ -160,6 +160,7 @@ cli_cmd_ta_brick_parse(const char **words, int wordcount, char **ta_brick)
         goto out;
     }
 
+    *ta_brick[0] = 0;
```

Comment:
Perhaps just replace all those strcat() and all with a single simple snprintf?

Reply:
Thank you @mykaul, it was replaced